README.md

```diff
@@ -53,7 +54,8 @@ gulp.task("publish", function () {
                     // use gzip for assets that benefit from it
                     gzip: true,
                     // cache static assets for 20 years
-                    cacheTime: 630720000
+                    cacheTime: 216000                     
```

Comment:
Sorry about this. I haven't seen this comment until now.
The number of `cacheTime` and `sharedCacheTime` are different to show the case when you want  longer cache time on the CDN. This is usually good because you can invalidate cache manually on the CDN.